app/register/page.tsx

```diff
@@ -56,15 +56,14 @@ const Register = (): JSX.Element => {
     if (isSignedIn) {
       router.push('/weeklyPicks');
     }
-  }, [isSignedIn]);
+  }, [isSignedIn, router]);
 
   const form = useForm<RegisterUserSchemaType>({
     resolver: zodResolver(RegisterUserSchema),
   });
 
   /**
    * The current value of the 'email' field in the form.
-   *
```

Comment:
Why are these being removed? This was created to separate the description from the types.

Reply:
This was removed automatically on save; must be a setting with the eslint and prettier setup.

Reply:
Updated in latest commit.